GG/src/Layout.cpp

```diff
@@ -512,10 +489,20 @@ void Layout::Render()
 {
     if (m_render_outline) {
         Pt ul = UpperLeft(), lr = LowerRight();
-        FlatRectangle(ul, lr, CLR_ZERO, m_outline_color, 1);
-        for (const std::vector<Rect>& columns : CellRects()) {
-            for (const Rect& cell : columns) {
-                FlatRectangle(cell.ul, cell.lr, CLR_ZERO, m_outline_color, 1);
+        FlatRectangle(ul, lr, CLR_ZERO, CLR_MAGENTA, 1);
+        auto cell_rects{CellRects()};
```

Comment:
What is the type of this? `CellRects`? Why not:

```suggestion
        CellRects cell_rects{};
```

Reply:
This is a call to `std::vector<std::vector<Rect>> CellRects() const;`